app.js

```diff
@@ -1,173 +1,102 @@
 var hours = ['10AM', '11AM', '12AM', '1PM', '2PM', '3PM', '4PM', '5PM',];
 
-var stores = [
-{
-  name: 'Pike Place',
-  min: 17,
-  max: 88,
-  avgSales: 5.2,
-  hourlySales: [],
-  totals: 0,
-  grandTotal: 0,
-  customerPerHour: function () {
+var pikePlace = new Store('Pike Place', 17, 88, 5.2);
```

Comment:
'Typically' you would not put these up at the top of the file for human-readability purposes. It's tough for another dev to read this and know what the `new Store` object actually is. At the least, go ahead and put them below your constructor and prototype methods.